src/nginx/varify/varify.go

```diff
@@ -54,3 +74,24 @@ func main() {
 		log.Fatalf("Could not write config file: %s", err)
 	}
 }
+
+func readNameServers(resolvConfPath string, defaultNameServer string) ([]string, error) {
+	resolvConfFile, err := os.Open(resolvConfPath)
+	if err != nil {
+		return []string{defaultNameServer}, err
```

Comment:
Go convention is for the first value to be empty when the error is non-nil, and for the consumer to treat it as undefined.